paddle/fluid/operators/activation_op.cc

```diff
@@ -458,21 +458,22 @@ namespace ops = paddle::operators;
 
 #define FOR_EACH_INPLACE_OP_FUNCTOR(__macro) \
   __macro(Sigmoid, sigmoid);                 \
-  __macro(Relu, relu);                       \
   __macro(Exp, exp);                         \
-  __macro(Tanh, tanh);                       \
   __macro(Ceil, ceil);                       \
   __macro(Floor, floor);                     \
-  __macro(Sqrt, sqrt);                       \
   __macro(SoftRelu, soft_relu);              \
   __macro(Relu6, relu6);                     \
   __macro(Reciprocal, reciprocal);           \
   __macro(HardSigmoid, hard_sigmoid);
 
+#define FOR_EACH_MKLDNN_INPLACE_OP_FUNCTOR(__macro) \
```

Comment:
Why changes of `activation_op.cc`?
- Are the previous registration not worked? But I see that the unit-test `test_activation_mkldnn_op.py` works well.
- Current changes are not suitable for other devices. For example, if there is `amd_relu_op`, how should it be defined?